sources/web/datalab/static/appbar.html

```diff
@@ -12,12 +12,12 @@
     <span class="autosave_status"></span>
   </span>
   <div class="btn-toolbar pull-right right-toolbar">
-    <div class="btn-group">
-      <button id="ungitButton" title="Open ungit on the notebooks repository" class="toolbar-btn">
+    <div id="ungitButton" class="btn-group">
```

Comment:
Here is a SO article with some discussion that mentions accessibility and screen readers as reasons not to us a div as a button: http://stackoverflow.com/questions/2777256/making-buttons-button-or-div